### ISSUE TITLE
Fix jsx-a11y/accessible-emoji console error

### ISSUE THREAD

**Issue by Dishebh** [fossasia/pslab-desktop#601]
* **I'm submitting a ...**
  - [x] bug report
  - [ ] feature request

* **What is the current status?**
Fix console errors for FAQ.js file,

./src/screen/FAQ/FAQ.js
`Line 439:  Emojis should be wrapped in <span>, have role="img", and have an accessible description with aria-label or aria-labelledby  jsx-a11y/accessible-emoji`

* **If the current behavior is a bug, please provide the steps to reproduce it**
> A screenshot of the issue may be sufficient for UI bugs

* **What is the expected behavior?**
Emojis should be wrapped around span as mentioned in the error.

* **Please tell us about your environment:**
  - OS with version number: [ Windows / OSX / Linux Distro ]

* **Other information** ( Any research that you may have done which you think is the cause of problem )

* **Would you like to work on it?**
Yes
